.github/workflows/cryo_gen.yml

```diff
@@ -9,6 +9,7 @@ on:
       - 'openfasoc/common/**'
       - 'openfasoc/generators/common/**'
       - 'openfasoc/generators/common/cryo-gen/**'
```

Comment:
These `generators/common/[generator]` paths can be removed. They were supposed to be `generators/[generator]` which I missed in my PR.